static-assets/components/cstudio-forms/controls/rte.js

```diff
@@ -152,6 +156,17 @@ CStudioAuthoring.Module.requireModule(
 										manager.awaitRteConfigInitialization(store, doRteInitialization);
 										manager.dispatchInitRTEConfig(store);
 									});
+
+								// If ui config XML not loaded yet, dispatch action to load it
+								const state = store.getState();
+								if (state && !state.uiConfig.xml) {
```

Comment:
I don't state can ever be nullish if there's a store already. UI config could, though.
```suggestion
								if (!state.uiConfig?.xml) {
```